docs-site/content/0.19.0/guide/features/faceting.md

```diff
@@ -0,0 +1,338 @@
+
+<Tabs :tabs="['JavaScript','PHP','Python','Ruby']">
+  <template v-slot:JavaScript>
+
+```javascript
+let searchParameters = {
+  'q'         : 'experiment',
+  'query_by'  : 'title',
+  'facet_by' : 'ratings_count',
+  'sort_by'   : 'average_rating:desc'
+}
+
+client.collections('books')
+  .documents()
+  .search(searchParameters)
+  .then(function (searchResults) {
+    console.log(searchResults)
+  })
+```
+  </template>
+
+  <template v-slot:PHP>
+
+```php
+$searchParameters = [
+  'q'         => 'experiment',
+  'query_by'  => 'title',
+  'facet_by'  => 'ratings_count',
+  'sort_by'   => 'average_rating:desc'
+]
+
+$client->collections['books']->documents->search($searchParameters)
+```
+  </template> 
+  <template v-slot:Python>
+
+```python
+search_parameters = {
+  'q'         : 'experiment',
+  'query_by'  : 'title',
+  'facet_by' : 'ratings_count',
+  'sort_by'   : 'average_rating:desc'
+}
+
+client.collections['books'].documents.search(search_parameters)
+```
+   </template>
+   <template v-slot:Ruby>
+
+```ruby
+search_parameters = {
+  'q'         => 'experiment',
+  'query_by'  => 'title',
+  'facet_by'  => 'ratings_count',
+  'sort_by'   => 'average_rating:desc'
+}
+
+client.collections['books'].documents.search(search_parameters)
+```
+  </template>
+</Tabs>
+
+Smaple response:
+
+```json
+{
+   "facet_counts": [
+      {
+         "counts": [
+            {
+               "count": 1,
+               "highlighted": "6058",
+               "value": "6058"
+            },
+            {
+               "count": 1,
+               "highlighted": "28177",
+               "value": "28177"
+            },
+            {
+               "count": 1,
+               "highlighted": "172302",
+               "value": "172302"
+            }
+         ],
+         "field_name": "ratings_count",
+         "stats": {
+            "avg": 68845.66666666667,
+            "max": 172302.0,
+            "min": 6058.0,
+            "sum": 206537.0
+         }
+      }
+   ],
+   "found": 3,
+   "hits": [
+      {
+         "document": {
+            "authors": [
+               "James Patterson"
+            ],
+            "authors_facet": [
+               "James Patterson"
+            ],
+            "average_rating": 4.08,
+            "id": "569",
+            "image_url": "https://images.gr-assets.com/books/1339277875m/13152.jpg",
+            "publication_year": 2005,
+            "publication_year_facet": "2005",
+            "ratings_count": 172302,
+            "title": "The Angel Experiment"
+         },
+      }
+      ...
+   ]
+}
+```
+
+Note the additional `stats` field in the result that shows the avg, min, max and sum value for `ratings_count` field. You can limit the number of faceted results using `max_facet_value` parameter. More details on these arguments can be found [here](https://typesense.org/docs/0.19.0/api/documents.html#arguments).
```

Comment:
For links within the docs site could you use relative links, so Vuepress doesn't open the link in a new page: 

```
[here](../../api/documents.html#arguments)
```